### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by JOHN-72** [community/community#36339] [Other Feature Feedback, Questions, & Ideas] [answered]
does only initiating a pull work will count in progress or they need to be merged too ?

**Comment by GR8xlr8** [comment]
I also have similar kind of doubt can anybody please clarify this

**Comment by arana2k** [comment, marked as answer]
You need to get them merged 
as it been stated by them 
"Have 4 pull/merge requests accepted between October 1 and October 31 to complete Hacktoberfest"
so you need to have your pr merged by the contributer

**Comment by akasyaotomotiv** [comment]
@abhishek8778

**Comment by ossanna16** [comment]
Hey there @JOHN-72 and thank you for asking a good question! I'm glad you found our community 🙂
